test/hotspot/jtreg/gc/g1/numa/TestG1NUMATouchRegions.java

```diff
@@ -33,7 +33,7 @@
  *          java.management
  * @build sun.hotspot.WhiteBox
  * @run driver jdk.test.lib.helpers.ClassFileInstaller sun.hotspot.WhiteBox
- * @run main/othervm -XX:+UseG1GC -Xbootclasspath/a:. -XX:+UseNUMA -XX:+UnlockDiagnosticVMOptions -XX:+WhiteBoxAPI gc.g1.TestG1NUMATouchRegions
+ * @run main/othervm -XX:+UseG1GC -Xbootclasspath/a:. -XX:+UseNUMA -XX:+UnlockDiagnosticVMOptions -XX:+WhiteBoxAPI gc.g1.numa.TestG1NUMATouchRegions
```

Comment:
Since you are touching this overlong line anyway (and making it even longer), could you add a line break or two in there?

Reply:
fixed!